components/esm/loadcell.cpp

```diff
@@ -176,10 +176,20 @@ bool Cell::getNextRef(ESMReader &esm, CellRef &ref, bool& deleted)
 
     // NOTE: We should not need this check. It is a safety check until we have checked
     // more plugins, and how they treat these moved references.
-    if (esm.isNextSub("MVRF")) {
-        // skip rest of cell record (moved references), they are handled elsewhere
-        esm.skipRecord(); // skip MVRF, CNDT
-        return false;
+    if (esm.isNextSub("MVRF"))
+    {
+        if (ignoreMoves)
```

Comment:
Isn't this naming backwards? <i>ignoreMoves</i> would suggest when the parameter is true, the MVRF record is skipped, but the code does the opposite.